### PR TITLE
added back button and created back button component

### PR DESCRIPTION
**Pull request by ariasamandi**
Kinda forgot the format for pull requests and im lazy.

I added a component for back button and a link to go back to the dashboard, and added the image in image folder.

### CODE REVIEW

**Comment by ariasamandi** [comment]
Also learned while trying to get the css from figma to work that apparantly hyphens do not work. Thanks to chatgpt for explaining this: 

"The syntax errors you're encountering in the code you provided are due to the fact that you are using CSS property names with hyphens inside an object literal in JavaScript. In JavaScript object literals, property names cannot contain hyphens. To use such property names, you should wrap them in quotes as strings."

Basically just added quotes around the css property names that had hyphens such as: 'flex-shrink': `0`,
